versions/tasks.py

```diff
@@ -43,16 +49,20 @@ def import_versions(delete_versions=False, new_versions_only=False, token=None):
     client = GithubAPIClient(token=token)
     tags = client.get_tags()
 
+    import_version_task_group = []
     for tag in tags:
         name = tag["name"]
 
         if skip_tag(name, new_versions_only):
             continue
 
         logger.info("import_versions_importing_version", version_name=name)
-        import_version.delay(name, tag=tag, token=token)
+        import_version_task_group.append(import_version.s(name, tag=tag, token=token))
 
-    # Get all release notes
+    task_group = group(*import_version_task_group)
+    if purge_after:
+        task_group.link(purge_fastly_release_cache.s())
```

Comment:
Creates a group and chains the purge command to run after all of the `import_version_task_group` tasks have completed